sources/Interop/Windows/shared/winerror/Windows.Manual.cs

```diff
@@ -963,361 +961,12902 @@ public static partial class Windows
 
         public const int ERROR_NO_SUCH_DEVICE = 433;
 
+        public const int ERROR_CAPAUTHZ_NOT_DEVUNLOCKED = 450;
+
+        public const int ERROR_CAPAUTHZ_CHANGE_TYPE = 451;
+
+        public const int ERROR_CAPAUTHZ_NOT_PROVISIONED = 452;
+
+        public const int ERROR_CAPAUTHZ_NOT_AUTHORIZED = 453;
+
+        public const int ERROR_CAPAUTHZ_NO_POLICY = 454;
+
+        public const int ERROR_CAPAUTHZ_DB_CORRUPTED = 455;
+
+        public const int ERROR_CAPAUTHZ_SCCD_INVALID_CATALOG = 456;
+
+        public const int ERROR_CAPAUTHZ_SCCD_NO_AUTH_ENTITY = 457;
+
+        public const int ERROR_CAPAUTHZ_SCCD_PARSE_ERROR = 458;
+
+        public const int ERROR_CAPAUTHZ_SCCD_DEV_MODE_REQUIRED = 459;
+
+        public const int ERROR_CAPAUTHZ_SCCD_NO_CAPABILITY_MATCH = 460;
+
+        public const int ERROR_PNP_QUERY_REMOVE_DEVICE_TIMEOUT = 480;
+
+        public const int ERROR_PNP_QUERY_REMOVE_RELATED_DEVICE_TIMEOUT = 481;
+
+        public const int ERROR_PNP_QUERY_REMOVE_UNRELATED_DEVICE_TIMEOUT = 482;
+
+        public const int ERROR_DEVICE_HARDWARE_ERROR = 483;
+
+        public const int ERROR_INVALID_ADDRESS = 487;
+
+        public const int ERROR_VRF_CFG_ENABLED = 1183;
+
+        public const int ERROR_PARTITION_TERMINATING = 1184;
+
+        public const int ERROR_USER_PROFILE_LOAD = 500;
+
         public const int ERROR_ARITHMETIC_OVERFLOW = 534;
 
-        public const int SEVERITY_SUCCESS = 0;
```

Comment:
I wonder if we could do something to help minimize this diff...

This moved down to https://github.com/terrafx/terrafx.interop.windows/pull/43/files#diff-902801b79342b8c581d349536688d080R6432, so maybe if we inserted 5464 lines in a commit before the commit containing the new defines....